integration/src/test/resources/org/mule/test/construct/flow.xml

```diff
@@ -33,8 +33,8 @@
 
     <flow name="flow2">
         <composite-source>
-            <custom-source class="org.mule.test.construct.FlowConfigurationFunctionalTestCase$TestSimpleMessageSource"/>
-            <custom-source class="org.mule.test.construct.FlowConfigurationFunctionalTestCase$TestSimpleMessageSource"/>
+            <test:custom-source class="org.mule.test.construct.FlowConfigurationFunctionalTestCase$TestSimpleMessageSource"/>
```

Comment:
can you put the logic from this source into `SkeletonSource` to avoid having this class?